### ISSUE TITLE
NoSQL database integration

### ISSUE THREAD

**Issue by supersergiy** [ZettaAI/zetta_utils#136]

**Comment by supersergiy** [comment]
```python
l = build_kv_layer(backend=DataStoreBackend(), ....)

Read:
l[a: str] -> get the ‘value’ column for key “a”
l[(a: str, b: Tuple[str, ...])] -> get “b” columns for key “a” 
l[list[a: str], b: Tuple[str, ...]] -> batch read

write:
l[a: str] = value_BLOB
l[(a: str, b: Tuple[str, ...])] = {}
l[list[a: str], b: Tuple[str, ...]] = [{}]
```

**Comment by supersergiy** [comment]
Closed via #157
